src/test/event-source-mapping-status-advisor.test.ts

```diff
@@ -0,0 +1,54 @@
+import { Worker } from "../core";
+import { EventSourceMappingStatusAdvisor } from "../core/event-source-mapping-status-advisor";
+
+describe("App", () => {
+  it("should not contain suggestion 'Event Source Mapping should be enabled' when integration has no properties", () => {
```

Comment:
Recheck this [KMS-SQS](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-sqs-queue.html#cfn-sqs-queue-kmsmasterkeyid)